### PR TITLE
Add Findora Chain

### PR DESCRIPTION
**Pull request by zqsun**
This PR adds the details for the Findora Chain to chainlist. Findora Chain is an EVM chain with privacy features.

### CODE REVIEW

File: _data/chains/eip155-2152.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://findora.org/",
+  "shortName": "fra",
+  "chainId": 2152,
+  "networkId": 2152,
+  "explorers": [
+    {
+      "name": "findorascan",
+      "url": "https://evm.findorascan.io/",
```
Comment (ligi) [inline _data/chains/eip155-2152.json:19]:
```suggestion
      "url": "https://evm.findorascan.io",
```

File: _data/chains/eip155-2153.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://findora.org/",
+  "shortName": "fra",
+  "chainId": 2153,
+  "networkId": 2153,
+  "explorers": [
+    {
+      "name": "findorascan",
+      "url": "https://testnet-anvil.evm.findorascan.io/",
```
Comment (ligi) [inline _data/chains/eip155-2153.json:21]:
```suggestion
      "url": "https://testnet-anvil.evm.findorascan.io",
```

**Review by ligi — CHANGES_REQUESTED** [review]
shortNames must be unique

**Review by DONALDSULLENDER — APPROVED** [review]

**Review by zqsun — COMMENTED** [review]
Update the testnet short name

**Pull request closed without merging**